### DISCUSSION TITLE
Vercel vitals and Cross-Origin-Resource-Policy

### DISCUSSION THREAD

**Discussion by gregermendle** [vercel/community#1270] [Help] [answered]
### Summary

Hey!

So I'm working on a project that utilizes ffmpeg.wasm. This requires setting the following headers to get access to `SharedArrayBuffer` in Chrome, Firefox and friends.

```
Cross-Origin-Embedder-Policy: require-corp
Cross-Origin-Opener-Policy: same-origin
```

The issue I'm running into is with Vercel Anaytics. It seems that there is no `Cross-Origin-Resource-Policy: cross-origin` on any of the responses from vercel.vitals, etc. This makes it so that I can no longer use Vercel analytics on my site. Google analytics works, as well as fonts etc, since they supply that header. 

Is there anyway around this? Am I just doing something wrong?

Thanks for any help!

**Comment by amyegan** [comment, marked as answer]
Thanks for reporting this issue @gregermendle. I let the analytics engineers know, and they will look into it 🙇

**Reply by amyegan** [reply]
The analytics team just updated the API to include cross-origin headers 🎉

**Reply by bergold** [reply]
I have the same issue, but with the _Vercel Toolbar_. It's loaded from 
https://vercel.live/_next-live/feedback/feedback.js and doesn't include the `Cross-Origin-Resource-Policy: cross-origin` header in its response.

Is it possible to also include the header there, so that we can keep using preview comments on our product?

Thank you for any help!

**Reply by barziahmed** [reply]
@amyegan I have the same issue as @bergold .  I cannot use Vercel Toolbar at all because of CORP policy. Please prioritize this.

**Reply by dohooo** [reply]
The same @amyegan

**Comment by returningsam** [comment]
Hello! I am also attempting to use `SharedArrayBuffer` in my project and am running into this same issue.

I am getting errors on all requests for `vitals.vercel-insights.com`.

Is there anything I can do to make sure vercel vitals work on my project?

Thanks in advance!
